### ISSUE TITLE
My Account area - navigation tab isn't active when adding new payment method 

### ISSUE THREAD

**Issue by adriandmitroca** [woocommerce/woocommerce#25039]
**Describe the bug**
Navigation tab is not set as active when adding new payment method.

**To Reproduce**
1. Go to my account area
2. Go to payment methods tab
3. Try to enter new payment method
4. On your right, you will see "Payment methods" tab not highlighted

**Expected behavior**
"Payment methods" tab should be highlighted.

**WordPress Environment**
WordPress: 5.3
WooCommerce: 3.8.0
WooCommerce Stripe Gateway: 4.3.1

Basically, you can simply fix this by attaching to a filter, but suppose this issue should be addressed to the core as well. Please see example below.

```php
add_filter('woocommerce_account_menu_item_classes', function ($classes, $endpoint) {
	if ($endpoint === 'payment-methods' && isset($GLOBALS['wp']->query_vars['add-payment-method'])) {
		$classes[] = 'is-active';
	}

	return $classes;
}, 1, 2);
```

**Comment by claudiosanches** [comment]
Thank you for reporting this issue!

I'm closing the issue as it already has a PR, please continue the discussion over at #25041.
